site/styles/_mixins.scss

```diff
@@ -77,7 +77,6 @@
   &:hover,
   &:focus {
     color: var(--link-color);
-    text-decoration: none;
```

Comment:
This seems like a rather small change, but why? This doesn't appear to be the main point of this PR.